person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..500)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    age_of_person = of_age?
+    (return true if age_of_person || @parent_permission)
+  end
+
+  private
+
+  def of_age?
+    (@age >= 18)
+    true
+  end
```

Comment:
- [ ] Your `can_use_services?` always returns `true` regardless of the `age` of the person or `parent_permission`. In the image below age is less than 18 and parent_permission is set to false yet it returns true. This is because in Ruby there is an implicit return of the last statement in a method. **`of_age?`** has `true` as the last statement hence it will always return true, which also affects `can_use_services?` Kindly fix this.

  You can resolve this by simply having `of_age?` return only `@age >= 18` (which returns true or false) and `can_use_services?` return `of_age? || @parent_permission` (which returns true if any of the values is true, otherwise false)

  ![alex-can-use](https://user-images.githubusercontent.com/36542986/150898318-593d05e6-8e34-4a91-88e6-78f65cf9425e.jpg)

Reply:
Hi @TedLivist thanks so much for your suggestion, I'll fix it now! 😃